diff --git a/dist/index.js b/dist/index.js
index 565796cc999fd3088e73568781850270c5e56a10..0a842f5168a0c6e615610d3bcb4b5dfe1979f1ad 100644
--- a/dist/index.js
+++ b/dist/index.js
@@ -1,4 +1,4 @@
-import{set as e,unset as n,defineType as i,definePlugin as t}from"sanity";import{jsx as o,jsxs as r}from"react/jsx-runtime";import{EllipsisHorizontalIcon as c,TrashIcon as l,SyncIcon as a}from"@sanity/icons";import{Card as d,Flex as s,Box as u,Text as p,Popover as m,Button as g,Menu as h,MenuItem as f,Autocomplete as y}from"@sanity/ui";import{useState as C,useEffect as w,useMemo as b,useCallback as k}from"react";import*as I from"lucide-react";const v=({selectedIcon:e,onReplace:n,onClear:i,readOnly:t=!1})=>{const[y,b]=C(!1);return w(()=>{const e=e=>{"Escape"===e.key&&y&&b(!1)};return y&&document.addEventListener("keydown",e),()=>{document.removeEventListener("keydown",e)}},[y]),/* @__PURE__ */o(d,{border:!0,padding:1,radius:2,tone:"default",onClick:()=>{t||n()},children:/* @__PURE__ */r(s,{align:"center",justify:"space-between",children:[
+import{set as e,unset as n,defineType as i,definePlugin as t}from"sanity";import{jsx as o,jsxs as r}from"react/jsx-runtime";import{EllipsisHorizontalIcon as c}from"@sanity/icons/EllipsisHorizontal";import{TrashIcon as l}from"@sanity/icons/Trash";import{SyncIcon as a}from"@sanity/icons/Sync";import{Card as d,Flex as s,Box as u,Text as p,Popover as m,Button as g,Menu as h,MenuItem as f,Autocomplete as y}from"@sanity/ui";import{useState as C,useEffect as w,useMemo as b,useCallback as k}from"react";import*as I from"lucide-react";const v=({selectedIcon:e,onReplace:n,onClear:i,readOnly:t=!1})=>{const[y,b]=C(!1);return w(()=>{const e=e=>{"Escape"===e.key&&y&&b(!1)};return y&&document.addEventListener("keydown",e),()=>{document.removeEventListener("keydown",e)}},[y]),/* @__PURE__ */o(d,{border:!0,padding:1,radius:2,tone:"default",onClick:()=>{t||n()},children:/* @__PURE__ */r(s,{align:"center",justify:"space-between",children:[
 /* @__PURE__ */r(s,{align:"center",gap:2,padding:2,children:[
 /* @__PURE__ */o(u,{style:{fontSize:"1.25rem",marginBottom:"-0.25rem",lineHeight:1},children:e?.icon?/* @__PURE__ */o(e.icon,{}):/* @__PURE__ */o(p,{size:1,muted:!0,children:"?"})}),
 /* @__PURE__ */o(u,{children:/* @__PURE__ */o(p,{size:1,weight:"medium",children:e?.label||"Unknown icon"})})]}),!t&&/* @__PURE__ */o(m,{content:/* @__PURE__ */r(h,{children:[
diff --git a/src/LucideIconPicker.tsx b/src/LucideIconPicker.tsx
index eb551fe0c7a1b3d76cb86cd1d1c75e33b03ba296..e0e0e0bdc57ac7dcafa68fd45c9e339e1470d43d 100644
--- a/src/LucideIconPicker.tsx
+++ b/src/LucideIconPicker.tsx
@@ -1,4 +1,4 @@
-import { TrashIcon } from '@sanity/icons';
+import { TrashIcon } from '@sanity/icons/Trash';
 import { Autocomplete, Box, Button, Card, Flex, Text } from '@sanity/ui';
 import { type JSX, useCallback, useEffect, useMemo, useState } from 'react';
 import { type StringInputProps, set, unset } from 'sanity';
diff --git a/src/SelectedIconCard.tsx b/src/SelectedIconCard.tsx
index 7de71460d054cbe427278cf37fc23231b579951b..fa603b380469f6596bf6f470dcba0c7ddccc2d40 100644
--- a/src/SelectedIconCard.tsx
+++ b/src/SelectedIconCard.tsx
@@ -1,4 +1,6 @@
-import { EllipsisHorizontalIcon, SyncIcon, TrashIcon } from '@sanity/icons';
+import { EllipsisHorizontalIcon } from '@sanity/icons/EllipsisHorizontal';
+import { SyncIcon } from '@sanity/icons/Sync';
+import { TrashIcon } from '@sanity/icons/Trash';
 import {
   Box,
   Button,
